common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4216,6 +4206,42 @@ object CelebornConf extends Logging {
       .stringConf
       .createOptional
 
+  val QUOTA_DISK_BYTES_WRITTEN: ConfigEntry[Long] =
+    buildConf("celeborn.quota.diskBytesWritten")
```

Comment:
Shall we add a tenant field for these configurations? such as celeborn.quota.tenant.diskBytesWritten, to be clear these configurations are for multi-tenancy

Reply:
> Shall we add a tenant field for these configurations? such as celeborn.quota.tenant.diskBytesWritten, to be clear these configurations are for multi-tenancy

But system also use this....

Reply:
> > Shall we add a tenant field for these configurations? such as celeborn.quota.tenant.diskBytesWritten, to be clear these configurations are for multi-tenancy
> 
> But system also use this....

These system level configuration is also for tenants(not for an unique tenant)

Reply:
> > > Shall we add a tenant field for these configurations? such as celeborn.quota.tenant.diskBytesWritten, to be clear these configurations are for multi-tenancy
> > 
> > 
> > But system also use this....
> 
> These system level configuration is also for tenants(not for an unique tenant)

Ok， done